apps/beeswax/src/beeswax/server/dbms.py

```diff
@@ -389,6 +429,19 @@ def get_query_server_config_via_connector(connector):
       'QUERY_TIMEOUT_S': 15 * 60,
   }
 
+  if 'service' in connector['options']:
+    server_config['crto_hs2_service'] = connector['options']['service']
+
+  if 'dc' in connector['options']:
+    server_config['crto_dc'] = connector['options']['dc']
+
+  if 'env' in connector['options']:
+    server_config['crto_env'] = connector['options']['env']
+
+  server_config['crto_connector_type'] = connector['type']
```

Comment:
This should not be used, but I keep around in case we switch to using connectors